### PR TITLE
feat: add Toggle Airplane Mode & INetworkActions Interface

### PR DESCRIPTION
**Pull request by akinsolb**
## Change list

Please provide briefly described change list which are you going to propose. 
 
## Types of changes

What types of changes are you proposing/introducing to .NET client?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] New feature (non-breaking change which adds value to the project)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Documentation
- [x] Have you proposed a file change/ PR with appium to update documentation? 
## Integration tests
- [x] Have you provided integration tests to pass against the beta version of appium? (for Bugfix or New feature)

## Details

Add support for toggling Airplane mode on Android

Please provide more details about changes if it is necessary. If there are new features you can provide code samples which show the way they
work and possible use cases. Also you can create [gists](https://gist.github.com) with pasted C# code samples or put them here using markdown. 
About markdown please read [Mastering markdown](https://guides.github.com/features/mastering-markdown/) and [Writing on GitHub](https://help.github.com/categories/writing-on-github/)

### CODE REVIEW

File: src/Appium.Net/Appium/Android/AndroidDriver.cs [hunk 1]
```diff
@@ -191,11 +191,18 @@ public ConnectionType ConnectionType
         public void LongPressKeyCode(int keyCode, int metastate = -1) =>
             AppiumCommandExecutionHelper.LongPressKeyCode(this, keyCode, metastate);
 
+        #region Device Network
+
+        public void ToggleAirplaneMode() => AndroidCommandExecutionHelper.ToggleAirplaneMode(this);
```
Comment (mykola-mokhnach) [inline src/Appium.Net/Appium/Android/AndroidDriver.cs:196]: Please add a docstring. Also, this endpoint only works in Android below version 7

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs:196]: Is, "Only supports below API 24 (Android 7)" good enough. Btw the xml document inherits from the interface

Comment (mykola-mokhnach) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs:196]: yes, should be good.
Also, it probably, makes sense to mention that on emulator that works in any version. Toggling data transfer works ONLY on emulator.

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs:196]: Are you sure about that? I just ran the ToggleAiplaneMode test on an Android 8.1 emulator (api 27) and receive the below
```
An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/Users/X/Library/Android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell am broadcast -a android.intent.action.AIRPLANE_MODE --ez state false' exited with code 255'; Stderr: 'Security exception: Permission Denial: not allowed to send broadcast android.intent.action.AIRPLANE_MODE from pid=29600, uid=2000
```

Initially I ran tests on an Android 9 physical device

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs:196]: As for the toggling data, you mean that Toggling Airplane mode (back on) will turn on everything but mobile data on real devices?

Comment (mykola-mokhnach) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs:199]: I've double checked the server code. You're right, on emulator the same command is used as for a real device, so it should also fail. Sorry, for the confusion. I've probably mixed it with toggleData.

The toggleData stuff enables/disables data transfer on emulators. There is no API to do this via command line on real devices though

Comment (akinsolb) [reply src/Appium.Net/Appium/Android/AndroidDriver.cs:199]: Ok. i've updated the docs for toggleData since I moved it to the interface

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**